protocol/x/clob/keeper/keeper.go

```diff
@@ -262,6 +263,33 @@ func (k Keeper) InitializeNewStreams(ctx sdk.Context) {
 				clobPairId,
 			)
 		},
+		func(subaccountId satypes.SubaccountId) *satypes.StreamSubaccountUpdate {
+			subaccount := k.subaccountsKeeper.GetSubaccount(
```

Comment:
nit: could we make a funciton on 
```
// Subaccount defines a single sub-account for a given address.
// Subaccounts are uniquely indexed by a subaccountNumber/owner pair.
type Subaccount struct {
```
that goes `subaccount.ToStreamSubaccountUpdate()`?

Reply:
I'm actually doing this in a followup PR/branch already, will address this there.